framework/stats/stats_provider.cpp

```diff
@@ -0,0 +1,53 @@
+/* Copyright (c) 2020, Broadcom Inc. and Contributors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 the "License";
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "stats_provider.h"
+
+namespace vkb
+{
+// Default graphing values for stats. May be overridden by individual providers.
+std::map<StatIndex, StatGraphData> StatsProvider::def_graph_map{
```

Comment:
Nitpick but could this be renamed `default_graph_map`?

Reply:
Yep - that was a bit lazy on my part :-)